### PR TITLE
feat(app): Add website manifest, minify CSS for production

### PR DESCRIPTION
**Pull request by EugeneHlushko**
- Add website manifest and respective icons
- Copy assets into ./dist when building the website

### CODE REVIEW

File: src/server.jsx [hunk 1]
```diff
@@ -48,6 +48,7 @@ export default locals => {
           { Object.keys(assets).filter(asset => /\.css$/.test(asset)).map(path => (
             <link key={ path } rel="stylesheet" href={ `/${path}` } />
           ))}
+          <link rel="manifest" href="/manifest.json" />
```
Comment (anshumanv) [inline src/server.jsx]: Will this be able to read manifest in dev as well? 🤔

Comment (EugeneHlushko) [reply src/server.jsx]: No this file is only used for prod builds. On dev we are using HTML plugin for the HTML boilerplate.

**Comment by montogeek** [comment]
This is nice to have, but do we need it? How many users are installing PWA?

File: src/server.jsx (hunk 1 above)
Comment (anshumanv) [reply src/server.jsx]: Sounds good! 💯

**Comment by EugeneHlushko** [comment]
Its very nice to have for cases when you are commuting and losing connectivity. I am looking at service worker implementation now, so will continue in further PRs when have time

**Comment by EugeneHlushko** [comment]
oops forgot to mention @montogeek ;]

**Comment by EugeneHlushko** [comment]
Generally my idea is to get all pages work offline, there a way to get to that point, but its possible.

**Comment by montogeek** [comment]
I guess it it nice to have :)
We need to make sure though to publicize it, tell users that it is now possible to read the docs offline, maybe a bagde or one time notification.

**Comment by EugeneHlushko** [comment]
Yeah we can, but when its finished ;]

**Comment by montogeek** [comment]
Thanks!

**Comment by pranshuchittora** [comment]
Awesome :rocket: :clap:

**Pull request merged**